common-design-patterns/signature.md

```diff
@@ -0,0 +1,147 @@
+---
+altLangPage: "https://conception.canada.ca/configurations-conception-communes/fil-ariane.html"
+breadcrumbs:
+  - title: "About Canada.ca"
+    link:  "https://www.canada.ca/en/government/about.html"
+  - title: Canada.ca design system
+    link: "https://www.canada.ca/en/government/about/design-system.html"
+  - title: Template and pattern library
+    link: "https://www.canada.ca/en/government/about/design-system/pattern-library.html"    
+  - title: Global header
+    link: "https://design.canada.ca/common-design-patterns/global-header.html"
```

Comment:
@delisma -  Breadcrumbs are generated by new config, but we don't have a good solution for children yet